.gitignore

```diff
@@ -131,3 +131,6 @@ tests/functional/custom_java_predictor/**/*.jar
 # misc
 .metals
 .vscode
+
+# used to download large LLM models, library is not published yet
+!datarobot_custom_code-0.0.1-py3-none-any.whl
```

Comment:
github repo [is here](https://github.com/datarobot/datarobot-custom-code)

Reply:
@elatt , it's a separate repo and It's not really ready to be published as a library anywhere.
So I built a wheel and included it into the NIM drop-in environment.

Reply:
We can't merge this branch like this though.

Reply:
Maybe like Trition, we need to do two phases, checkin first w/o this tool and have follow on work to add it later.

Reply:
Removed the wheel library, I will add the source code into the dropin environment instead.

Reply:
Yeah I think this is better. Just make this code part of drum and we can split it out if others want to use it.